src/FermentableDialog.cpp

```diff
@@ -47,14 +49,16 @@ FermentableDialog::FermentableDialog(MainWindow* parent) :
    tableWidget->setModel(fermTableProxy);
    tableWidget->setSortingEnabled(true);
    tableWidget->sortByColumn( FERMNAMECOL, Qt::AscendingOrder );
+   tableWidget->verticalHeader()->setDefaultSectionSize(20);
```

Comment:
You should remove this line. This should not be handled here but in FermentableTableModel.
And I quite sure that it is not a good idea to have a fixed value because it could leads to display bugs in some graphical environment.